_config.yml

```diff
@@ -76,8 +76,8 @@ footer:
         url: https://plus.google.com/u/0/b/116512812300813784482/116512812300813784482
             
     links:
-      - label: Freenode
-        url: http://webchat.freenode.net/?channels=google-containers
+      - label: Slack
+        url: https://kubernetes.slack.com
```

Comment:
They need to sign up at http://slack.k8s.io/, so probably we should link to that instead